code/object/objectshield.cpp

```diff
@@ -17,34 +17,69 @@
 
 #include <limits.h>
 
+// Private variables
+static double factor = 1.0 / (log(50.0) - log(1.0));	// Factor used in Goober5000's scale_quad
```

Comment:
Also, this variable seems to be a constant so it should be declared `const` to make sure no one changes it.